Existing patch from the X gate to resolve compile errors due to
Bug 20376077 - arc4random prototypes added only to non-__STDC__ portion of <stdlib.h>

Need to determine if still needed.

--- a/Key.c	Mon Mar 23 09:21:31 2015
+++ b/Key.c	Mon Mar 23 11:20:24 2015
@@ -36,6 +36,8 @@
 #include <bsd/stdlib.h> /* for arc4random_buf() */
 #endif
 
+extern void arc4random_buf();
+
 #ifndef HAVE_ARC4RANDOM_BUF
 static void
 getbits (long data, unsigned char *dst)
